packages/gateway/src/rate-limit.test.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+
+import { resolvers as rootResolvers } from '@gateway/features/user/root-resolvers'
+import { resolvers as locationRootResolvers } from '@gateway/features/location/root-resolvers'
+import * as fetchAny from 'jest-fetch-mock'
+import * as jwt from 'jsonwebtoken'
+import { readFileSync } from 'fs'
+// eslint-disable-next-line import/no-relative-parent-imports
+import { clearRedisMock } from '../test/setupJest'
+
+const fetch = fetchAny as any
+const resolvers = rootResolvers as any
+const locationResolvers = locationRootResolvers as any
```

Comment:
could look into getting rid of "any"s

Reply:
The resolvers require a "context" -param which requires a full "Request" -param which is generated by Hapi. Not gonna fix in this PR, requires thought into how we'd wanna mock the requests.